Dockerfile

```diff
@@ -168,6 +168,17 @@ RUN curl -fsSL "$NODE_DOWNLOAD_URL" -o node.tar.gz \
   && yarn install \
   && rm -rf /node.tar.gz package.json yarn.lock /tmp/* ~/.npm
 
+### Elixir + Sobelow
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends wget gnupg gnupg1 gnupg2 \
+    && wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb \
```

Comment:
@alex0112 Just a quick check, I am not super familiar but do we need to pull from packages.erlang-solutions.com rather than https://www.erlang.org/ or something.